_studio/mfx_lib/encode_hw/h264/src/mfx_h264_encode_hw.cpp

```diff
@@ -3500,7 +3513,12 @@ mfxStatus ImplementationAvc::AsyncRoutine(mfxBitstream * bs)
 
                 mfxStatus sts = MFX_ERR_NONE;
 
-                //printf("Execute: %d, type %d, qp %d\n", task->m_frameOrder, task->m_type[0], task->m_cqpValue[0]);
+                //printf("Execute: %d, type %d, qp %d\n", task->m_frameOrder, task->m_type[0], task->m_cqpValue[0]);#if defined(MXF_ENABLE_MCTF_IN_AVC)
```

Comment:
Why do you add "#if defined(MXF_ENABLE_MCTF_IN_AVC)" in the end of line here?

Reply:
Sorry this is a typo, I did not realize I put it on that line instead of the next one, I will fix this